### PR TITLE
Enable configuration of http proxy

### PR DESCRIPTION
**Pull request by martinberanek**
### Requirements

* Enable configuration of http proxy via ansible

### Description of the Change

* Inspired by https://github.com/ansible-ThoTeam/nexus3-oss/blob/master/files/groovy/setup_http_proxy.groovy

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -32,6 +32,23 @@ nexus_max_filedescriptors: 65536
 nexus_admin_password: admin123
 nexus_anonymous_access: true
 
+# HTTP
+nexus_http_proxy_host: ""
+nexus_http_proxy_port: 3128
+nexus_http_proxy_username: ""
+nexus_http_proxy_password: ""
+
+nexus_https_proxy_host: ""`
```
Comment (jnogol) [inline defaults/main.yml]: There's an extra ` in here.

File: tasks/config_nexus.yml [hunk 2]
```diff
@@ -67,6 +68,9 @@
   with_items:
     - "{{ nexus_scheduled_tasks }}"
 
+- name: NEXUS | System -> Http
+  import_tasks: action/system/http.yml
+
```
Comment (jnogol) [inline tasks/config_nexus.yml:73]: IMO, there should be a variable to toggle this action

Comment (martinberanek) [reply tasks/config_nexus.yml:73]: And what should this variable toggle? The logic is in groovy file and IMHO it's better to run task every time.

File: defaults/main.yml (hunk 1 above)
Comment (martinberanek) [reply defaults/main.yml]: fixed

**Comment by jnogol** [comment]
Could you please pull changes from develop? They fix the Travis tests.

**Comment by martinberanek** [comment]
Rebased

**Comment by martinberanek** [comment]
@jnogol Will you accept the pull request? Or should I change something else?

**Pull request closed without merging**